### DISCUSSION TITLE
Is there a way to configure per-domain zoom levels?

### DISCUSSION THREAD

**Discussion by ghjardim** [qutebrowser/qutebrowser#6201] [Q&A] [answered]
Hi! I'd want to configure qutebrowser to have 100% zoom level for all domains, except for colab.research.google.com, which I'd like to be 75% by default. I remember that other browsers, such as Chromium, do have support to this feature.

**Comment by The-Compiler** [comment, marked as answer]
Not yet - see #3636 and #3782.
